src/Extensions/Compatability/Validation/NotNullWhenAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#if NETFRAMEWORK || NETSTANDARD2_0
+namespace System.Diagnostics.CodeAnalysis
+{
+	/// <summary>Specifies that when a method returns <see cref="ReturnValue"/>, the parameter will not be null even if the corresponding type allows it.</summary>
```

Comment:
`<summary>` tags should be on their own lines

Reply:
Fixed, thanks